### PR TITLE
OH-11 | Update pharmaceutical form

### PR DESCRIPTION
**Pull request by lenatoukam12**

### CODE REVIEW

File: src/main/java/org/isf/medicals/gui/MedicalEdit.java [hunk 1]
```diff
@@ -400,7 +422,52 @@ private VoLimitedTextField getCodeTextField() {
...
+	}
+	
+	/**
+	 * This method initializes dosingTextField
+	 * 
+	 * @return javax.swing.JTextField
+	 */
+	private VoLimitedTextField getDosingTextField() {
+		if (dosingTextField == null) {
+			dosingTextField = new VoLimitedTextField(5);
```
Comment (ArnaudFofou) [inline src/main/java/org/isf/medicals/gui/MedicalEdit.java]: please fix indentation here

File: src/main/java/org/isf/medicals/gui/MedicalEdit.java [hunk 2]
```diff
@@ -400,7 +422,52 @@ private VoLimitedTextField getCodeTextField() {
...
+	}
+	
+	/**
+	 * This method initializes conditioningTextField
+	 * 
+	 * @return javax.swing.JTextField
+	 */
+	private VoLimitedTextField getConditioningTextField() {
+		if (conditioningTextField == null) {
+				conditioningTextField = new VoLimitedTextField(5);
```
Comment (ArnaudFofou) [inline src/main/java/org/isf/medicals/gui/MedicalEdit.java]: fix indentation here

**Review by ArnaudFofou — APPROVED** [review]

**Review by JantBogard — APPROVED** [review]
LGTM

File: src/main/java/org/isf/medicals/gui/MedicalEdit.java (hunk 1 above)
Comment (ArnaudFofou) [inline src/main/java/org/isf/medicals/gui/MedicalEdit.java]:
```suggestion
			dosingTextField = new VoLimitedTextField(50);
```

File: src/main/java/org/isf/medicals/gui/MedicalEdit.java (hunk 2 above)
Comment (ArnaudFofou) [inline src/main/java/org/isf/medicals/gui/MedicalEdit.java]:
```suggestion
				conditioningTextField = new VoLimitedTextField(10);
```

**Review by ArnaudFofou — APPROVED** [review]

**Comment by ArnaudFofou** [comment]
LGTM. Thank!

**Pull request merged**